driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientVertex.java

```diff
@@ -269,6 +269,7 @@ public Object createLink(final ODocument iFromVertex, final OIdentifiable iTo, f
 
         return streamVertices.stream()
                 .flatMap(edges -> edges.stream())
+                .filter(oId -> oId != null)
```

Comment:
i tried, but it was too hard to isolate the problem, so i just added this line in the end :(